pennylane/tape/tape.py

```diff
@@ -653,6 +653,23 @@ def inv(self):
 
         self._ops = list(reversed(self._ops))
 
+    def adjoint(self):
+        new_tape = self.copy(copy_operations=True)
+        new_tape.inv()
+
+        # the current implementation of the adjoint
+        # method requires that the returned inverted object
```

Comment:
Are you referring to the qnode transform here?

Reply:
Yep! `qml.adjoint()` requires a `Operation.adjoint()` method be implemented on objects that it is recursively adjointing